data/job-board-rules-guidelines.md

```diff
@@ -0,0 +1,28 @@
+### 1. Job Submission and Approval Process:
+
+- After successful job submission, each job will undergo a verification process by the PyCon India 2023 team.
+- The approval process may take approximately 4-5 business days.
+- Approved jobs will be made visible on the job board once they pass the verification process.
+- Submissions that do not meet the guidelines may be rejected, and submitters will be notified of the reasons for rejection.
```

Comment:
```suggestion
- After successful job submission, each job will undergo a verification process by the PyCon India 2023 team.
- The approval process may take approximately 4-5 business days.
- Approved jobs will be made visible on the job board once they pass the verification process.
- Submissions that do not meet the guidelines may be rejected, and submitters will be notified of the reasons for rejection.
- We do not impose any restrictions on the industry/technology of the submitted jobs. Any jobs that are helpful to the community are welcome.
- The job board will stay open for the community even after the conference is over. If the position you posted has been filled, and you are no longer taking applications, please let the team know, so we can take down your posting. 
```

@RajatRajdeep @bhansa Please review the last point once. Since we're keeping the job board even later. What's the process for taking down postings that are no longer required? Or should we just let them be?